satpy/composites/spectral.py

```diff
@@ -159,9 +159,9 @@ def __call__(self, projectables, optional_datasets=None, **attrs):
         LOG.info(f"Applying NDVI-weighted hybrid-green correction with limits [{self.limits[0]}, "
                  f"{self.limits[1]}] and strength {self.strength}.")
 
-        ndvi_input = self.match_data_arrays([projectables[1], projectables[2]])
+        ndvi_input = self.match_data_arrays(projectables)
```

Comment:
```suggestion
        projectables = self.match_data_arrays(projectables)
        ndvi_input = projectables[1:]
```

Reply:
Updated, but I simply used the `projectables` elements directly instead of the intermediate `ndvi_input`